### ISSUE TITLE
SA1013 Allow } to start a line in interpolated string

### ISSUE THREAD

**Issue by morsiu** [DotNetAnalyzers/StyleCopAnalyzers#3914]
Currently, SA1013 interferes in situations where I would like to split very long interpolated strings into multiple lines, or when tooling like Resharper does that when formatting code.
This can be seen in the following example code:
``` csharp
var message =
    $"This a very long message with id {
        id
    }, that is easier to read when split into multiple lines ({
        numberOfLines
    } lines). Some additional expression to make it even longer: {
        (foobarbaz * xyz) / (foobarbaz + xyz)
    }";
```
Here, SA1013 generates warnings on lines starting with `}` (i.e. lines 4, 6 and 8).
The warning is that `}` should not be preceded by whitespace.
My suggestion is to not generate these warning in this case.

**Comment by sharwell** [comment]
Resolving as wontfix as mentioned in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/pull/3915#issuecomment-3573162279
